internal/cephfs/volumeoptions.go

```diff
@@ -109,24 +110,30 @@ func extractMounter(dest *string, options map[string]string) error {
 	return nil
 }
 
-func getMonsAndClusterID(options map[string]string) (string, string, error) {
+func getClusterInformation(options map[string]string) (string, string, string, error) {
```

Comment:
Wow, returning three strings now, that becomes quite a lot. Maybe the `util.clusterInfo` struct should become usable. This can be done at a later time, in a separate PR (see #998).

Reply:
Yes, I had the same thought but didn't want to blow up the PR. I see you've already assigned someone but I could also have a go if you want me to.

Reply:
It is not urgent, so it really does not matter to me who works on it. Yug is a new colleague here at Red Hat and is expected to contribute more to Ceph-CSI in the future, so these tasks are nice for him to learn more about the project.